packages/react/src/components/OverflowMenu/OverflowMenu.tsx

```diff
@@ -347,11 +487,11 @@ class OverflowMenu extends Component {
       this.state.open &&
       (!this._menuBody || !this._menuBody.contains(evt.target))
     ) {
-      this.closeMenu();
+      this.closeMenu(); // Fix: Pass the onCloseMenu callback when calling closeMenu
```

Comment:
Is this fixed? Or an outdated comment maybe?

Reply:
Its an outdated comment, I had added for my own reference, forgot to remove it.